### ISSUE TITLE
Future updates to this add-on will be Nexus+ only

### ISSUE THREAD

**Issue by anxdpanic** [anxdpanic/plugin.video.youtube#413]

**Comment by Isayso** [comment]
AFAIK there is no stable Nexus versions for Raspberry systems, just beta versions with lots of problems. It would be nice at least to fix problems, like the slow download issues that arise sometimes. Thanks!

**Comment by greatquux** [comment]
> AFAIK there is no stable Nexus versions for Raspberry systems, just beta versions with lots of problems. It would be nice at least to fix problems, like the slow download issues that arise sometimes. Thanks!

Yeah we're following progress in https://forums.raspberrypi.com/viewtopic.php?p=2090599 but there are some issues building inputstream_adaptive.  Kodi 20 probably won't get into the Raspbian repo until it upgrades to the next major Debian release (bookworm) anyway.   So hopefully it doesn't break! :)

**Comment by lasciare393** [comment]
Kodi 19.x (Matrix) is already using Python 3. Why do you require Nexus since Matrix could be enough?

Nexus version is too recent and it'll take many years for it to propogate variety of linux distros, android boxes etc. I'd like to kindly ask to reconsider lowering it to Matrix.

**Comment by JimmyS83** [comment]
Ah, thats unfortunate :(  If anyone knows about project, which would backport most important updates (like actual 'video dont play in this app') to the k18/k19 branches, please let other know here..

**Comment by neo-neo1** [comment]
This sucks for many. Nexus 20 is not supported for those using CoreELEC, LibreELEC, OSMC & others on older hardware.
I'm affected on a 3 year old SBC hardware (not Rasp Pi). Like many, It doesn't have Nexus 20 support.

I guess beggars can't be choosers.

None the less, thanks for your continued maintenance of this irreplaceable project.

**Comment by Isayso** [comment]
> 
> None the less, thanks for your continued maintenance of this irreplaceable project.

Agree, thanks for this addon. If I had an alternative to Kodi, I would switch, spent countless hours in set-up, maintaining and  chasing bugs. With every major release the mess starts again. I tested LibreElec 11 Nexus this week, it is a downgrade in stability and functionality.....  very disappointing 

Thanks for your work!   

Update: Tested xbian with 20.1 on Rpi3+, much much better than Libre, no random audio stutter, no 60Hz problem, no strange addon problems with rumble and Disney. Hope for old devices ;)

**Comment by Basti83x** [comment]
Hello, do I understand that correctly?  there will be no update for kodi 19 in the future which will fix the current problem?

**Comment by ThubLives** [comment]
@Basti83x The main dev has warned that he is extremely busy leading up to the end of the month, which is an unfortunate coincidence that the current problem would kicked off when it did.  I'm sure he'd like to support previous versions, but right now we don't even have a satisfactory solution that works for everyone.  When that eventually happens, I suspect there are enough people running older Kodi versions that if @anxdpanic can't find the time to do it, someone else will hopefully step up and backport the fix to the older release for the older Kodi.

In short, we shall have to wait and see.

**Comment by Basti83x** [comment]
Ok. Thanks

**Comment by MoojMidge** [comment]
For the time being current releases, starting from [v7.0.2.2](https://github.com/anxdpanic/plugin.video.youtube/releases/tag/v7.0.2.2), will work in Kodi Matrix
